stac_mjx/compute_stac.py

```diff
@@ -42,17 +44,7 @@ def root_optimization(mjx_model, mjx_data, kp_data, frame: int = 0):
     qs_to_opt = jnp.zeros_like(q0, dtype=bool)
     qs_to_opt = qs_to_opt.at[:7].set(True)
     # kps_to_opt = jnp.repeat(jnp.ones(len(utils.params["kp_names"]), dtype=bool), 3)
```

Comment:
This is kind of unrelated, but do you mind adding a comment as to why we're keeping this around? (Eg, `# Enable for debugging blah blah...`)